Tests/SwiftSyntaxBuilderTest/StringInterpolationTests.swift

```diff
@@ -460,4 +460,22 @@ final class StringInterpolationTests: XCTestCase {
       )
     }
   }
+
+  func testCodeBlockItemListInterpolation() {
+    let codeBlockItemList: CodeBlockItemListSyntax =
+      """
+      let one = object.methodOne()
+      let two = object.methodTwo()
+      func foo() {}
+      """
+    XCTAssertTrue(codeBlockItemList.is(CodeBlockItemListSyntax.self))
+    XCTAssertEqual(
+      codeBlockItemList.description,
+      """
+      let one = object.methodOne()
+      let two = object.methodTwo()
+      func foo() {}
+      """
+    )
```

Comment:
We have an `Assert` that can be used instead. 
In it you can provide `codeBlockItemList` and a string with the expected string

Reply:
Sorry for not being clear enough. 
I was thing more like this `AssertBuildResult`
https://github.com/apple/swift-syntax/blob/main/Tests/SwiftSyntaxBuilderTest/ArrayExprTests.swift#L24

Reply:
Thanks for your comments! Since the ``StringInterpolationTests.swift`` is used for testing the *string interpolation*, I've add another class to test the build result in ``CodeBlockItemListTests.swift``. I wonder whether this is reasonable? @kimdv

Reply:
Both looks good for me. 😁